src/core/Form/Toggle/ToggleBase/ToggleBase.tsx

```diff
@@ -0,0 +1,37 @@
+import { ReactNode } from 'react';
+import { HtmlSpanProps } from '../../../../reset/HtmlSpan/HtmlSpan';
+
+export const baseClassName = 'fi-toggle';
+
+export interface ToggleBaseProps {
+  /** Controlled toggle-state - if provided, component will update only when this is explicitly changed */
+  checked?: boolean;
+  /** Default status of toggle when not using controlled 'checked' state
+   * @default false
+   */
+  defaultChecked?: boolean;
+  /** Custom classname to extend or customize */
+  className?: string;
+  /** Disable usage */
+  disabled?: boolean;
+  /**
+   * Label element content
+   */
+  children?: ReactNode;
+  /**
+   * aria-label for the HTML input-element,
+   */
+  'aria-label'?: string;
```

Comment:
We should update the JSDoc as now it says that `aria-label` is for input but it is used also for the **button** version.

Reply:
Changed it to "for toggle" instead. The other is passed on to button and the other on to input. Not sure if these should be there at all, as there is the "label" as ReactNode already.